roles/ceph-common/tasks/main.yml

```diff
@@ -191,7 +191,15 @@
     creates="{{ fetch_directory }}/ceph_cluster_uuid.conf"
   register: cluster_uuid
```

Comment:
I actually tested several times and it doesn't seem to be a problem. Here's what I did:

* initial bootstrap: fetch dir got created with proper values
* re provision again: no problem
* remove fetch dir and run provision: fetch gets created and ceph_cluster_uuid.conf with the right value